modules/openapi-generator/src/main/resources/csharp/ApiClient.mustache

```diff
@@ -150,13 +150,13 @@ namespace {{packageName}}.Client
         public ISerializer Serializer => this;
         public IDeserializer Deserializer => this;
 
-        public string[] AcceptedContentTypes => RestSharp.Serializers.ContentType.JsonAccept;
+        public string[] AcceptedContentTypes => RestSharp.ContentType.JsonAccept;
 
         public SupportsContentType SupportsContentType => contentType =>
-            contentType.EndsWith("json", StringComparison.InvariantCultureIgnoreCase) ||
-            contentType.EndsWith("javascript", StringComparison.InvariantCultureIgnoreCase);
+            contentType.Value.EndsWith("json", StringComparison.InvariantCultureIgnoreCase) ||
+            contentType.Value.EndsWith("javascript", StringComparison.InvariantCultureIgnoreCase);
 
-        public string ContentType
+        public ContentType ContentType
```

Comment:
Just remove the set. And do you really need the backing field? Just use `public ContentType ContentType { get; }`

Reply:
An ISerializer's ContentType requires a setter. Looks like the exception for the setter was added quite a while ago, as part of a whole refactor.
I think it may be better to do what RestSharp currently does for its JsonNetSerializer and do `public ContentType ContentType { get; set; } = ContentType.Json;`.